packages/apps/src/contexts/activity.ts

```diff
@@ -106,6 +123,12 @@ export interface IActivityContext<T extends Activity = Activity>
   signout: (name?: string) => Promise<void>;
 }
 
+const DEFAULT_SIGNIN_OPTIONS: ISignInOptions = {
+  connectionName: 'graph',
+  oauthCardText: 'Please Sign In...',
+  signInButtonText: 'Sign In',
```

Comment:
should we also add the ability for them to add other activity props if they want? Right now there is work being done by Leon where they want in BF to be able to trigger the sign in flow but add extra activity data points so exposing that as well would be helpful.

Reply:
Updated. Take a look. The isGroup clause is still curious though. What's the expected behavior there? User goes to their 1:1 chat, then grants SSO there?

Reply:
Should this be the regular card that we're sending to 1:1 anyway?

Reply:
ya, thats the hack GCP team came up with to support group chat oauth flows.